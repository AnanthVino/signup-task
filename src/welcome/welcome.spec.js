import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Welcome from '.';
    
describe('Welcome', () => {
  it('should render correctly', () => {
    
    const wrapper = shallow(<Welcome />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})