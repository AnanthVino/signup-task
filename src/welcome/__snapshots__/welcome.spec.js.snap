// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render correctly 1`] = `
<Fragment>
  <nav
    className="navbar navbar-expand-lg navbar-dark text-light bg-primary"
  >
    <h3>
      Ajar Online
    </h3>
  </nav>
  <div
    className="container d-flex flex-column content"
  >
    <p>
      Access Denied. You are not logged in.
    </p>
    <Link
      className="btn btn-info rounded-0"
      to="/"
    >
      Register
    </Link>
  </div>
</Fragment>
`;
